src/main/scala/org/encryfoundation/generator/Account.scala

```diff
@@ -0,0 +1,26 @@
+package org.encryfoundation.generator
+
+import java.net.InetSocketAddress
+import org.encryfoundation.common.Algos
+import org.encryfoundation.common.crypto.PrivateKey25519
+import scorex.crypto.hash.Blake2b256
+import scorex.crypto.signatures.{Curve25519, PrivateKey, PublicKey}
+import scala.io.Source
+
+case class Account(secret: PrivateKey25519, sourceNode: InetSocketAddress)
+
+object Account {
+
+  def parseFromFile(sourcePath: String): Seq[Account] = Source
+    .fromInputStream(getClass.getResourceAsStream(sourcePath))
+    .getLines()
+    .map { line =>
+      val splitLine: Array[String] = line.split("::")
+      val (privKey: PrivateKey, pubKey: PublicKey) = Curve25519.createKeyPair(
+        Blake2b256.hash(Algos.hash(splitLine.head + "mnemonic="))
```

Comment:
I think you should refactor it, so it won't throw NoSuchElementException in case of malformed file